samples/matter/lock/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf

```diff
@@ -34,3 +34,12 @@ CONFIG_IPC_RADIO_BT_HCI_IPC=y
 # that is pulled in by NETCORE_IPC_RADIO_IEEE802154 in application's Kconfig.sysbuild.
 # For Wi-Fi builds, this option will not get applied anyway.
 CONFIG_NRF_802154_ENCRYPTION=y
+
+# Do not generate or merge the MPI generated by the hci_ipc image.
+# It would result in the MPI configuration managed by the application core (main)
+# image to be overwritten by the default configuration from hci_ipc.
+CONFIG_SUIT_MPI_GENERATE=n
+CONFIG_SUIT_ENVELOPE_OUTPUT_MPI_MERGE=n
```

Comment:
Beware - the `west flash` has been broken and the MPIs are erased as part of `ERASE_UICR` operation.
To be on the safe side, you may want to keep the `CONFIG_SUIT_ENVELOPE_OUTPUT_MPI_MERGE` option enabled on radio core images.